seam/seam_webhook.py

```diff
@@ -4,10 +4,40 @@
 
 
 class SeamWebhook:
+    """
+    A class to handle Seam webhook verification.
+
+    This class provides functionality to verify incoming webhook payloads
+    using the Svix library.
+    """
+
     def __init__(self, secret: str):
+        """
+        Initialize the SeamWebhook instance.
+
+        Args:
+            secret (str): The secret key used for webhook verification.
```

Comment:
Where is this format taken from? If it's custom, let's instead adopt a standard format like [Sphinx](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format) that other tools are already able to parse.

Reply:
Nice, I'll adopt the standard format.

Reply:
https://github.com/seamapi/python/pull/113/commits/2782edde678e31c1b5e96edcd4ce88118a74465e